curator-framework/src/main/java/org/apache/curator/framework/imps/CreateBuilderImpl.java

```diff
@@ -485,7 +485,6 @@ private String protectedPathInForeground(String adjustedPath, byte[] data) throw
         }
         catch ( Exception e)
         {
-            ThreadUtils.checkInterrupted(e);
```

Comment:
The caller knows precisely _because_ InterruptedException is being thrown - that is how the information is conveyed to the caller. Check http://docs.oracle.com/javase/tutorial/essential/concurrency/interrupt.html:
> By convention, any method that exits by throwing an InterruptedException clears interrupt status when it does so.